sandbox_dns_unix.go

```diff
@@ -314,8 +315,15 @@ func (sb *sandbox) rebuildDNS() error {
 	dnsList = append(dnsList, resolvconf.GetNameservers(currRC.Content, types.IPv6)...)
 
 	// Resolver returns the options in the format resolv.conf expects
+	for _, option := range dnsOptionsList {
+		if strings.Contains(option, "ndots") {
+			goto build
```

Comment:
This assumes `ResolverOptions()` only returns the `ndots` option. While it is true now (we know it from the implementation, but not evident at all from the API), is this always going to be true in future ?
In case `dnsOptionsList` contains a user defined ndots option, I think we should instead prune any existing ndots option from the `sb.resolver.ResolverOptions()` list.